### PR TITLE
Update slip10

### PR DESCRIPTION
**Pull request by survived**
Warning should be nicely rendered on docs.rs, something like that: https://rustdoc.crud.net/imperio/warning-block/foo/struct.Foo.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # SLIP-10: Deterministic key generation
 
+<div class="warning">
+
+`slip-10` crate has been restructured and renamed into
+[`hd-wallet`](https://crates.io/crates/hd-wallet)! `slip-10` will not likely
```
Comment (maurges) [inline README.md]: s/will not likely/is not likely/

**Comment by maurges** [comment]
What about writing a migration guide? Shouldn't be too big. I can write it if you're busy

**Comment by survived** [comment]
We should probably put a guide into `m`, not into `legacy/slip-10` branch?

**Comment by maurges** [comment]
I think the guide should be in https://docs.rs/slip-10 and on the crates.io page, and github is not as important as it's already one repo

**Comment by survived** [comment]
Added a guide @maurges

**Comment by survived** [comment]
This is how it's rendered in docs: 

image

**Review by maurges — APPROVED** [review]

**Pull request merged**